testing/tests/DevExpress.ui.widgets.editors/dropDownBox.tests.js

```diff
@@ -441,7 +456,7 @@ QUnit.testInActiveWindow("first focusable element inside of content should get f
 
     keyboard.press("tab");
 
-    assert.equal($(instance.content()).parent(".dx-overlay-content").attr("tabindex"), -1, "popup content should not be tabbable");
+    assert.equal($(instance.content()).parent("." + OVERLAY_CONTENT_CLASS).attr("tabindex"), -1, "popup content should not be tabbable");
```

Comment:
strictEqual

Reply:
it is part of the old test, I only change the class name to the constant. It should be fixed in the separate PR and for the whole file